api/src/main/java/org/openmrs/module/reporting/common/SqlRunner.java

```diff
@@ -144,6 +144,54 @@ public SqlResult executeSql(String sql, Map<String, Object> parameterValues) {
         return result;
 	}
 
+    /**
+     * Executes a Sql Script
+     */
+    public SqlIterator executeSqlToIterator(String sql) {
+
+        SqlIterator iterator = null;
+        log.info("Executing SQL...");
+
+        List<String> sqlStatements = new ArrayList<String>();
+        sqlStatements.addAll(parseSqlIntoStatements(sql));
+
+        Boolean originalAutoCommit = null;
```

Comment:
Is there any reason why you are using `Boolean` instead of `boolean`?

Reply:
I'm fine with a Boolean